chapters/annotations.tex

```diff
@@ -68,27 +73,185 @@ \section{Annotations for Documentation}\doublelabel{annotations-for-documentatio
 \begin{lstlisting}[language=modelica]
 <a href="Modelica://MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
-\begin{lstlisting}[language=grammar]
-preferred-view-annotation:
-   annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+
+\subsection{Annotations for Figures\doublelabel{annotations-for-figures}}
+The \lstinline!figure-annotations! is a list of \lstinline!Figure! annotations given by:
+\begin{lstlisting}[language=modelica]
+record Figure
+  String title "Title meant for display";
+  String identifier "Identifier meant for programmatic access";
+  String group = "" "Name of plot group";
+  Boolean preferred = false "Automatically display figure after simulation";
+  Plot[:] plots "Plots";
+  String caption "Figure caption";
+end Figure;
+\end{lstlisting}
+
+A \lstinline!Figure! is a graphical container that can contain several
+\lstinline!plots! described by \lstinline!Plot! annotations:
+\begin{lstlisting}[language=modelica]
+record Plot
+  String title "Title meant for display";
+  String identifier "Identifier meant for programmatic access";
+  Curve[:] curves "Plot curves";
+  Axis x "X axis properties";
+  Axis y "Y axis properties";
+end Plot;
+\end{lstlisting}
+
+A \lstinline!Plot! can contain several \lstinline!curves!, see
+\autoref{plot-curves}, that all share a common x and y axis with properties
+described in \autoref{axis-properties}.
+
+The \lstinline!title! of a \lstinline!Figure! shall be non-empty and is
+mandatory, as every figure needs a title for presentation in contexts such as a
+list of figures, and as it is generally hard for tools to automatically generate
+a meaningful title.  On the other hand, when the \lstinline!title! of a
```

Comment:
The poll at the 101th design meeting determined that the `title` mustn't be required.  Instead, it should have a default value of `""`, where `""` means that the tool must produce a non-empty title based on the figure content.

Reply:
Addressed by #2580.